1-exercises/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,33 @@
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+
+arr.forEach(function(num){
```

Comment:
Could you indent the function inside forEach to make it more readable. A good example of indentation is the body of the function on line 24.